x-pack/plugins/actions/docs/openapi/components/schemas/config_properties_genai.yaml

```diff
@@ -0,0 +1,5 @@
+title: Connector request properties for a generative AI connector
+description: Defines properties for connectors when type is `.gen-ai`.
+type: object
```

Comment:
do you usually use just `object` here or should the required `properties` be present? 
```suggestion
type: object
properties:
    apiProvider:
        type: string
        description: The OpenAI API provider.
    apiUrl:
        type: string
        description: The OpenAI API endpoint.
```

Reply:
Ideally we'd have the individual properties, yes. Otherwise it's not too useful to our users and clients.